server/protocols/webadmin/webadmin-cassandra-data/src/test/java/org/apache/james/webadmin/routes/CassandraMappingsRoutesTest.java

```diff
@@ -81,7 +81,8 @@ void setUp(CassandraCluster cassandra) {
         webAdminServer = WebAdminUtils.createWebAdminServer(
             new DefaultMetricFactory(),
             new CassandraMappingsRoutes(cassandraMappingsService, taskManager, jsonTransformer),
-            new TasksRoutes(taskManager, jsonTransformer));
+            new TasksRoutes(taskManager, jsonTransformer))
+            .start();
```

Comment:
Indent

Reply:
I don't see much issue with this. What would you expect?

Reply:
the 4 lines with the new could be one indent more, so that they are not at the same indent level than the .start()